pubsub/README.md

```diff
@@ -112,30 +115,34 @@ message Message {
 }
 ```
 
-The `from` field denotes the author of the message, note that this is not
-necessarily the peer who sent the RPC this message is contained in. This is
-done to allow content to be routed through a swarm of pubsubbing peers.
+The `optional` fields may be omitted, depending on the
+[signature policy](#message-signing) and
+[message ID function](#message-identification).
 
-The `data` field is an opaque blob of data, it can contain any data that the
-publisher wants it to.
+The `from` field denotes the author of the message. This is the peer who
+initially authored the message, and NOT the peer who propagated it. Thus, as
+the message is routed through a swarm of pubsubbing peers, the original
```

Comment:
pubsubbing? I think one `b` might be more correct.

Reply:
Leaving aside that this verb doesn't exist in any English dictionary (😄), -ed and -ing forms of verbs that end in consonant + vowel + consonant duplicate the last consonant (stopped, dubbed, clubbed, batted, zapping, etc.), so this is correct.